### DISCUSSION TITLE
Artifacts on Halos after Restart

### DISCUSSION THREAD

**Discussion by tomgowan** [ufs-community/ufs-weather-model#1199] [Q&A]
I am modifying UFS restart files by replacing UFS cloud and ice condensates with cloud and ice condensates from another source. When I restart the run with this new data, I often see artifacts that appear on what I think are the halos of the sub-domains. Does anyone have any idea why this may be occurring? Is there some sort of adjustment or correction that occur on the halos when performing restarts? Below is an extreme example of these artifacts that shows up in the temperature field.

<img width="1539" alt="Screen Shot 2022-04-21 at 11 14 02 AM" src="https://user-images.githubusercontent.com/40776654/165829357-a3d3900d-f4bf-4d54-907d-92e9118d59b4.png">

**Comment by ShawnCebulaNOAA** [comment]
Hello @tomgowan!

Are you still seeing this issue?  I can check with the team if you still need help!  Sorry for the delay in checking into this!
Shawn Cebula - EPIC End User Support
